content/docs/custom-commands/commands.md

```diff
@@ -242,6 +252,61 @@ The modal trigger is used to trigger custom commands via submitting a modal.
 
 The trigger is matched using [RegEx](/docs/reference/regex).
 
+##### Crontab
+
+This trigger will run the command at scheduled intervals using [Cron Job Scheduling](https://en.wikipedia.org/wiki/Cron). This allows for advanced specificity in scheduling execution, such as scheduling execution at 23:45 every Saturday.
+
+When using an interval trigger, the custom command does not receive any user or member context. Thus, `{{ .User.ID }}`
+and similar templates will result in no value and member-dependent functions such as `addRoleID` will fail.
+
+![Overview of crontab configuration options.](crontab_trigger_options.png?width=60vw)
+
+<center>
+
+**1** Cron Expression **2** Channel **3** Excluding hours/weekdays
+
+</center>
+
+Cron Expression (**1**) defines the expression used to schedule the cron job. It uses the standard expression format (`minute hour day month dayofweek`). It does not support predefined schedules such as `@hourly`. The cron scheduler uses UTC always.
+
+Each field additionally supports special characters: `* / , -`. To read more about use of special characters, visit [Robfig's Cron package documentation - Special Characters](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-Special_Characters).
```

Comment:
I wonder if we should copy the relevant snippets of documentation there to save an extra click for interested users, with appropriate credit. WDYT @l-zeuch?

Reply:
For the record, these "special" characters are part of any decent cron implementation—the `cron` packages does nothing new here and ALLBUT the slash are also in the relevant standard.
This text is, in a similar way, also already on Wikipedia, just quite far down.

Copy from either the cron package or Wikipedia and link to the relevant section.